### ISSUE TITLE
buttons wont work 

### ISSUE THREAD

**Issue by Keratinx** [Lachee/discord-rpc-csharp#125]
i want to add buttons to my rich presence but it keeps bring a syntax error saying
 **'discordrpc.button' does not contain a definition for 'label'**

and same goes with the link, ive read the docs and even at some point copy the exact code from the repo and it still doesnt work for me

**Comment by Lachee** [comment]
are you able to share the code you are trying to use?

**Comment by CedricTorrekens** [comment]
Same issue here

**Comment by CedricTorrekens** [comment]
Discord doesn't let you click your own buttons. However, everyone else can. This is a limitation with Discord.

**Comment by Keratinx** [comment]
> Discord doesn't let you click your own buttons. However, everyone else can. This is a limitation with Discord.

yeah no my issue was with actually starting the program... it says its a syntax like it wasn't a function even though i updated it to the latest version.

**Comment by Lachee** [comment]
nobody is going to share their code so I can tell them what they are doing wrong? 
ok, I might as well close this issue then.

Issue templates exist for a reason.
